clients/rospy/src/rospy/topics.py

```diff
@@ -1236,7 +1236,7 @@ def get_impl(self, reg_type, resolved_name):
         elif reg_type == Registration.SUB:
             rmap = self.subs
         else:
-            raise TypeError("invalid reg_type: %s"%s)
+            raise TypeError("invalid reg_type: %s"%reg_type)
```

Comment:
```suggestion
            raise TypeError("invalid reg_type: %s"%(reg_type,))
```
I'd suggest adding the brackets to make the argument a 1-element tuple. It it most probably not needed here, but it is a good Python habit.